utils/SnapPy/eemepModelRunner

```diff
@@ -146,11 +151,15 @@ class EemepModelRunner():
             except:
                 fh. write("  started by: unknown\n")
 
+
+        xml_filename = Path(infile).name
+
         try:
             self._log(dirpath, 'Starting run for: {}'.format(infile))
-            mr = ModelRunner(dirpath, hpc, npp)
+            run_npp = True if xml_filename == self.npp_xml_filename else False
```

Comment:
`run_npp = xml_filename == self.npp_xml_filename`

Reply:
Please use brackets here, to make operator precedence clearer.